CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ set(CMAKE_MACOSX_RPATH 1)
 OPTION(JDK "Enable Java build")
 OPTION(CO_PILOT "Enable co-pilot build")
 OPTION(FORCE_32 "Force a 32bit compile on 64bit" OFF)
+option(USE_MUSL_LIBC "Use musl's libc implementation during compilation" OFF)
```

Comment:
Should this be all CAPS?

Reply:
I don't think it matters to CMake but I will change it for consistency.